src/app/examples/list/no-shape/list-no-shape-example.component.html

```diff
@@ -0,0 +1,26 @@
+<h2>List with no background shape:</h2>
+<h3>Removes background, border, shadow and padding from the list and the list item containers to provide flexibility
+</h3>
+<kirby-list [items]="items" shape="none" hasItemSpacing="true">
+
+  <kirby-card *kirbyListItem="let item">
+    <kirby-card-header *ngIf="item.amount < 0" title="Account is overdraft"></kirby-card-header>
+    <kirby-list-flex-item hasPadding="true">
```

Comment:
`hasPadding` property doesn't exist on `kirby-list-flex-item` - move up to `kirby-card` element

Reply:
Removed